blank-canvas-blocks/experimental-theme.json

```diff
@@ -161,6 +175,15 @@
 					"thickness": "2px",
 					"width": "150px"
 				},
+				"search": {
+					"button": {
+						"lineHeight": "27px",
```

Comment:
I think I understand why this is needed stylistically, but feels opaque to me. Like, I need to set the line height and padding on the button in order to control its overall height...

Reply:
yeah, and the icon is not going to be the same height as the text, so that's why we need to streamline it. We could do it in the CSS but it's needed for both Seedlet and BCB. so I didn't know of a more elegant way to do it.

Reply:
Rather than enforcing a line-height of a configured value to bring the icon into alignment, setting the line-height when an icon is used seems to do the same job without the need for tweaking to match.